contracts/staking/contracts/src/interfaces/IStakingPoolRewardVault.sol

```diff
@@ -20,189 +20,62 @@ pragma solidity ^0.5.9;
-    /// @param operatorBalance Balance in ETH of the operator.
-    /// @param membersBalance Balance in ETH co-owned by the pool members.
-    struct Pool {
-        bool initialized;
-        uint32 operatorShare;
-        uint96 operatorBalance;
-        uint96 membersBalance;
-        address payable operatorAddress;
-    }
-
-    /// @dev Emitted when the eth vault is changed
-    /// @param newEthVault address of new rth vault.
-    event EthVaultChanged(
-        address newEthVault
-    );
-
-    /// @dev Emitted when reward is deposited.
-    /// @param poolId The pool the reward was deposited for.
-    ///               Note that a poolId of "0" means "unknown" at time of deposit.
-    ///               In this case, the reward would be deposited later in the transaction.
-    ///               This is an optimization for the staking contract, which may make many deposits
-    ///               in the same transaction.
-    /// @param amount The amount in ETH deposited.
-    event RewardDeposited(
-        bytes32 poolId,
+    /// @dev Emitted when Eth is deposited into the vault.
+    /// @param sender Address of sender (`msg.sender`).
+    /// @param poolId that owns of Eth.
+    /// @param amount of Eth deposited.
+    event EthDepositedIntoVault(
+        address indexed sender,
+        bytes32 indexed poolId,
         uint256 amount
     );
 
-    /// @dev Emitted when a reward is withdrawn for an operator.
+    /// @dev Emitted when a reward is transferred to the ETH vault.
     /// @param amount The amount in ETH withdrawn.
+    /// @param member of the pool.
     /// @param poolId The pool the reward was deposited for.
-    event RewardWithdrawnForOperator(
-        bytes32 poolId,
-        uint256 amount
-    );
-
-    /// @dev Emitted when a reward is withdrawn for a pool member.
-    /// @param amount The amount in ETH withdrawn.
-    /// @param poolId The pool the reward was deposited for.
-    event RewardWithdrawnForMember(
+    event PoolRewardTransferredToEthVault(
         bytes32 poolId,
+        address member,
         uint256 amount
     );
 
-    /// @dev Emitted when a staking pool is registered.
-    /// @param poolId Unique Id of pool that was registered.
-    /// @param operatorShare Share of rewards owned by operator. in ppm.
-    event StakingPoolRegistered(
-        bytes32 poolId,
-        uint32 operatorShare
-    );
-
-    /// @dev Emitted when a staking pool's operator share is decreased.
-    /// @param poolId Unique Id of pool that was registered.
-    /// @param oldOperatorShare Previous share of rewards owned by operator.
-    /// @param newOperatorShare Newly decreased share of rewards owned by operator.
-    event OperatorShareDecreased(
-        bytes32 poolId,
-        uint32 oldOperatorShare,
-        uint32 newOperatorShare
+    /// @dev Emitted when the eth vault is changed
+    /// @param newEthVault address of new Eth vault.
+    event EthVaultChanged(
```

Comment:
Our convention elsewhere is `EthVaultSet`

Reply:
This event has been dropped.